index.js

```diff
@@ -128,11 +129,12 @@ async function createSentryRelease({ pluginApi, release, sentryEnvironment, sour
   await cli.releases.execute(['releases', 'deploys', release, 'new', '-e', sentryEnvironment])
 }
 
-async function createSentryConfig({ sentryOrg, sentryProject, sentryAuthToken }) {
+async function createSentryConfig({ sentryOrg, sentryProject, sentryAuthToken, sentryUrl }) {
   const sentryConfigFile = `
   [auth]
   token=${sentryAuthToken}
   [defaults]
+  url=${sentryUrl}
```

Comment:
@baffo since this a template string, we can use a conditional to determine whether to include this line if not null

Reply:
Thank you for the idea, I commited a conditional instead.